src/test/java/org/jenkinsci/plugins/workflow/cps/replay/ReplayActionTest.java

```diff
@@ -200,7 +201,7 @@ public static void assertPermissionId() {
                 List<Permission> permissions = Run.PERMISSIONS.getPermissions();
                 assertThat(permissions, Matchers.hasItem(ReplayAction.REPLAY));
                 gmas.add(ReplayAction.REPLAY, "dev2");
-                gmas.add(Jenkins.READ, "dev3");
+                gmas.add(Jenkins.Rit gEAD, "dev3");
```

Comment:
Did the cat step on your keyboard? :)

Reply:
@rsandell Um.... 🐱

Reply:
Interesting that he can almost spell _git_ correctly :smirk_cat: